zkfacade/src/main/java/com/yahoo/vespa/curator/Curator.java

```diff
@@ -201,7 +204,11 @@ public InterProcessLock createMutex(String lockPath) {
     // To avoid getting warning in log, see ticket 6389740
```

Comment:
Remove/edit this? Also, rename method to not include "Fake" since it is no longer fake?

Reply:
Hehe, good pointds.